chronos/asyncproc.nim

```diff
@@ -0,0 +1,1323 @@
+#
+#         Chronos' asynchronous process management
+#
+#  (c) Copyright 2022-Present Status Research & Development GmbH
+#
+#                Licensed under either of
+#    Apache License, version 2.0, (LICENSE-APACHEv2)
+#                MIT license (LICENSE-MIT)
+{.push raises: [Defect].}
+
+import std/[options, strtabs]
```

Comment:
we can use `Opt` instead, avoiding the need for this import (and the global symbols like `some` that it adds)

Reply:
done